engine/gamesys/proto/gamesys/gui_ddf.proto

```diff
@@ -215,7 +215,7 @@ message SceneDesc
         required string particlefx  = 2 [(resource)=true];
     }
 
-    required string          script               = 1 [(resource)=true];
+    optional string          script               = 1 [(resource)=true];
```

Comment:
we don't require `gui_script` on each `gui` component. So I changed it to be able to use protobuf metadata as is